ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/types.ts

```diff
@@ -3,3 +3,21 @@ export type PassingRateData = {
     passing: number;
     link: string;
 };
+
+interface ComplianceScanStatsShim {
```

Comment:
Why does the type name end in "Shim"?

Reply:
Not entirely sure I pulled it from the [proto](https://github.com/stackrox/stackrox/blob/master/proto/api/v2/compliance_results_service.proto#L49)